easy-ui-react/src/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useRef } from "react";
+import { useButton } from "react-aria";
+import { Icon } from "../Icon";
+import { ButtonColor } from "../Button";
+import { Text } from "../Text";
+import { IconSymbol } from "../types";
+import { classNames, variationName } from "../utilities/css";
+import { logWarningIfInvalidColorVariantCombination } from "../Button/utilities";
+import styles from "./IconButton.module.scss";
+
+export type IconButtonVariant = "filled" | "outlined";
+
+export type IconButtonProps = {
+  /** Button color */
+  color?: ButtonColor;
+  /** Button variant */
+  variant?: IconButtonVariant;
+  /** Icon symbol */
+  iconSymbol: IconSymbol;
```

Comment:
**thought:** based on other comment, curious about changing this property name to just `icon`?